### PR TITLE
Release v1.9.6

### PR DESCRIPTION
**Pull request by philipjohn**

### CODE REVIEW

**Review by rebeccahum — DISMISSED** [review]

**Review by GaryJones — DISMISSED** [review]

**Comment by okupdatenumber** [comment]
My Dear Developer 👨‍💻👨‍💻👨‍💻

Please Fix error 

I have tried several times.  Transferred these files to the World Press but still the live blog do not work there. 

Because , The plugin becomes active but it is not being enabled in the WP Post. error is currently showing.  Please do it right.
IMG_20200328_121202

**Comment by okupdatenumber** [comment]
A few years ago it worked correctly, but now due to an error, the entire plugin is not working

**Comment by GaryJones** [comment]
@okupdatenumber Please open a new issue, with as much detail as possible - what WP version? What Liveblog version? Was it working correctly with a previous version? Have you disabled other plugins and custom code / used a fresh install to show it's not a conflict? Give us the minimum setup needed to be able to reproduce the issue ourselves, so that we can then try and debug and find a solution.

**Comment by okupdatenumber** [comment]
https://github.com/Automattic/liveblog/issues/656#issue-589942960

**Comment by philipjohn** [comment]
There have been further merges to master since this PR was opened, so I've rebased and updated the changelog to reflect the other bug fixes that can now be included in v1.9.6

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
 
+## 1.9.6
+
+* Revert #597 (#639)
+* Harden check when rendering media library (#652)
+* Clean comment cache after direct SQL queries. (#658)
+* REST API routes require a permission_callback (#669)
+* Load CPT support later to avoid fatals with early use of WP_Query (#672)
+
+props [anigeluk], [david-binda], [GaryJones], [jeffersonrabb], [mslinnea], [philipjohn], [rebeccahum]
```
Comment (GaryJones) [inline CHANGELOG.md:11]: `anigeluk`, `jeffersonrabb`, and `mslinnea` all need corresponding links at the bottom of the file.

(There are also a couple of out-of-alphabetical-order ones at the very bottom - please move them to the correct order as well)

File: readme.txt [hunk 2]
```diff
@@ -3,8 +3,8 @@ Contributors: automattic, nbachiyski, batmoo, johnjamesjacoby, philipjohn
 Tags: liveblog
 Requires at least: 4.4
 Requires PHP: 5.6
-Tested up to: 4.9.8
-Stable tag: 1.9.5
+Tested up to: 5.3.2
```
Comment (GaryJones) [inline readme.txt:6]: Would be nice to have this tested on at least 5.7, if not a pre-5.8 release as well.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## 1.9.6
+
+* Revert #597 (#639)
```
Comment (GaryJones) [inline CHANGELOG.md:5]: Can we have a bit more detail on this one please? Someone reading this file not on GitHub, won't immediately have any sense of what was reverted.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,15 @@
...
+* Harden check when rendering media library (#652)
+* Clean comment cache after direct SQL queries. (#658)
```
Comment (GaryJones) [inline CHANGELOG.md:7]: This line has a full stop, but the others don't?

Comment (philipjohn) [reply CHANGELOG.md:7]: I am nothing if not incosistent. [a6f6e6ccd2c492a2e74d7dbe8043f1f08a6f7743]

File: readme.txt (hunk 2 above)
Comment (GaryJones) [reply readme.txt:6]: @philipjohn Has it been tested on latest WP?

**Comment by GaryJones** [comment]
Question on whether it has been tested on latest WP, and if so, could that `Tested up to` readme line be updated?

I also wonder whether we could do something like https://github.com/Parsely/wp-parsely/pull/187 first - GitHub action for auto-deploy to WordPress.org repo when adding a tag, readme.txt consolidated into README.md, screenshots numbering fixed and moved to `.wordpress-org` directory, changes extracted into CHANGELOG.md etc. All of these would ideally be done anyway, so doing it before this release just simplifies the release process (particularly the deploy to SVN).

Alternatively, if there's a time-pressure, then good to go on these 1.9.6 changes, and the other bits can be picked up before the next release.

File: readme.txt (hunk 2 above)
Comment (philipjohn) [reply readme.txt]: It's now been tested with 5.8 so I've updated the tested tag in [cd8d219]

**Comment by philipjohn** [comment]
Auto-deploying would be great, agreed. I'll work on that separately tomorrow!

**Comment by philipjohn** [comment]
FYI, auto-deployment in #675

**Comment by GaryJones** [comment]
At least one of the failing tests hasn't been touched for 5-9 years, so I think this logic was always flawed. The [previous Travis configuration](https://github.com/Automattic/liveblog/pull/674/files?file-filters%5B%5D=.dist&file-filters%5B%5D=.json&file-filters%5B%5D=.sh&file-filters%5B%5D=.yml#diff-6ac3f79fc25d95cd1e3d51da53a4b21b939437392578a35ae8cd6d5366ca5485) only tested up to PHP 7.0, so the issue was only revealed when the repo was switched to GitHub Actions, and up to PHP 7.4 was also tested.

**Pull request merged**